agents/ls-csharp/src/main/resources/org.eclipse.che.ls.csharp.script.sh

```diff
@@ -13,7 +13,7 @@ unset PACKAGES
 unset SUDO
 command -v tar >/dev/null 2>&1 || { PACKAGES=${PACKAGES}" tar"; }
 command -v curl >/dev/null 2>&1 || { PACKAGES=${PACKAGES}" curl"; }
-test "$(id -u)" = 0 || SUDO="sudo -E"
+test "$(id -u)" = 0 || test -f ${HOME}/is_arbitrary_user || SUDO="sudo -E"
```

Comment:
Yes of course that's what we've tried first. It did not work so we added this file to be able to skip `sudo`.

And to use existing images we would still need more work too. That's because in case of arbitrary user we need to modify `/etc/passwd/` and `/etc/group` as soon as the container is started https://github.com/redhat-developer/che-dockerfiles/blob/master/recipes/centos_jdk8/entrypoint.sh#L16.